src/DB/usersDB.mjs

```diff
@@ -38,13 +38,17 @@ export default async function (db) {
       applogger.trace('Searching for user "' + user._key)
       return user
     },
+    // NEW GET ROLE TYPES FUNCTION
     async getRoleTypes () {
       let query = 'FOR user IN users RETURN DISTINCT user.role'
       applogger.trace('Querying "' + query + '"')
       let cursor = await db.query(query)
       return cursor.all()
     },
+    // NEW GET USER FUNCTION
     async getUsers (countOnly, roleType, userEmail, sortDirection, offset, count) {
+      let queryString = ''
```

Comment:
'count' is a bit confusing IMHO (maybe it's word used in quasar?), what about 'limit'?

Reply:
It's what was used in TableAuditLogs, so we opted for consistency there. But the comment is valid, since the name doesn't really describe what the parameter entails. We can change it to `limit`, `rowLimit`, or even `rowsPerPage` (like it's called in the web component) across all table components.

Reply:
I like `rowsPerPage`. Can you also change it in the audit log please?